### DISCUSSION TITLE
Saber client 4 windows does not store server settings

### DISCUSSION THREAD

**Discussion by FantOEM** [saber-notes/saber#1285] [General]
Dear community,

I experience a problem with Windows Saber client. I can set it up and connect to nextcloud. Synchronization works. But with each restart of Saber, I am requested my nextcloud server settings (URL) and I have to re-confirm cloud access by logging into the nextcloud instance. It does not store URL or key.

How can I resolöve this behavior? On the MAC, this does not occur and both my iOS and Android mobiles work like a charm.

Any help is highly appreciated.

All the best,
FantOEM

**Comment by thomas-unk** [comment]
Same problem
Thomas

**Reply by FantOEM** [reply]
Any news from authors?
